lib/USAGE.md

```diff
@@ -101,21 +100,21 @@ Custom `Payload` or `Issuer` class can be created to customize their behavior.
 `Payload` provides hooks for adding functionality to the health card.
 
 ```ruby
-
 # Subclass the base `Payload` class to add specific behavior and/or set IG specific requirements
 class CustomHealthCard < HealthCards::Payload
   fhir_version '4.0.1' # Sets FHIR version
-  additional_types 'https://my.custom.cards#type' #Adds additional claim types to those required by SMART Health Cards
-  allow FHIR::Patient, %w[name] #Specify allowed attributes for FHIR resources
+  additional_types 'https://my.custom.cards#type' # Adds additional claim types to those required by SMART Health Cards
+  allow FHIR::Patient, %w[name] # Specify allowed attributes for FHIR resources
 end
+```
 
 ### Disable Public Key Resolution
 
 **Should the global configuration be prefixed with `globally_`? e.g. `HealthCards.globally_resolve_keys = false`**
 
 #### Globally
 Public key resolution can be configured across the library. This will affect the public key resolution
-for all classes and instances, including `Issuer` and `Payload
+for all classes and instances, including `Issuer` and `Payload`
 ```ruby
 # The current state can be checked with:
 HealthCards.resolves_keys?
```

Comment:
I was unable to get both `#resolves_keys?` and `#resolve_keys=` working for both `HealthCards` and `HealthCards::Verifier`. Any ideas for the best way to resolve this @radamson?

Reply:
`#resolve_keys` and `resolve_keys` are only `HealthCards::Verifier` instance methods. Setting key resolution at library and class level was something we were thinking about when `HealthCards` were responsible for more. So we can delete most of this except

```
Public key resolution can be disabled for a single instance issuer with:
```ruby
verifier = HealthCards::Verifier.new
# The current state can be checked with:
verifier.resolve_keys?
# Public key resolution can be disabled with
verifier.resolve_keys = false
# Keys can be manually added that the verifier can use to verify credentials
key = HealthCards::PrivateKey.from_file('my_keys.pem')
verifier.add_keys(key) # this only requires the public key so `verifier.add_keys(key.public_key)` works too
```
```

Reply:
Okay, I added a commit removing calls to all `resolve_keys` class methods, as well as the line mentioning that resolve_keys is supported by `HealthCard` in addition to `HealthCards::Verifier`.